src/command_modules/azure-cli-policyinsights/azure/cli/command_modules/policyinsights/commands.py

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+from ._client_factory import (policy_events_operations, policy_states_operations)
+from ._exception_handler import policy_insights_exception_handler
+
+
+def load_command_table(self, _):
+    policy_events_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.policyinsights.operations.policy_events_operations#PolicyEventsOperations.{}',
+        exception_handler=policy_insights_exception_handler
+    )
+
+    policy_states_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.policyinsights.operations.policy_states_operations#PolicyStatesOperations.{}',
+        exception_handler=policy_insights_exception_handler
+    )
+
+    with self.command_group('policy event', policy_events_sdk, client_factory=policy_events_operations) as g:
+        g.custom_command('list', 'list_policy_events')
```

Comment:
These commands are going to live side-by-side with the policy commands defined in `azure-cli-resource`. Is there a reason you don't just add your commands to that module?

Reply:
Yes, my intention is to maintain this package separately yet be part of the same command set. If the latter had not worked, I was planning to make it a separate command set.

Reply:
Where do/will the powershell commands reside? If they are separate from the rest of the `policy` commands then this is also fine for us @twitchax

Reply:
In PS, there is no command set concept. You can name your cmdlets the way you want following some naming conventions. We have 3 cmdlets in PS: Get-AzureRmPolicyEvent, Get-AzureRmPolicyState and Get-AzureRmPolicyStateSummary.

Reply:
I want to maintain the package separately for our RP. Not having to name the command set policyinsights makes it consistent with PS as well. This is probably not important for CLI, but I think it makes it much easier for customers to remember.

Reply:
We're fine with the new module as in powershell, the `policyinsights` folder is separate from that of `resources`